src/test/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.figure;
+
+public abstract class Figure {
+
+    public String color;
+    public String name;
+
+    public abstract void getDraw();
```

Comment:
you should create a separate interface FigureDrawer with method draw() that Figure should implement